recipes/_user.rb

```diff
@@ -20,6 +20,9 @@
 # Include the common recipe
 include_recipe 'omnibus::_common'
 
+# Install the omnibus toolchain so we have a shell
+include_recipe 'omnibus::_omnibus_toolchain' if omnibus_toolchain_enabled?
```

Comment:
To follow the idiom used in other recipes just move this guard to the `_omnibus_toolchain.rb` recipe:

```ruby
return unless omnibus_toolchain_enabled?
```

Ditto in `default.rb`: 
https://github.com/chef-cookbooks/omnibus/blob/9211e15caccba2ab4869852dd929e54e9c9fac28/recipes/default.rb#L24